fixtures/provider.py

```diff
@@ -366,3 +366,16 @@ def console_template(provider):
 @pytest.fixture(scope="module")
 def console_template_modscope(provider):
     return _get_template(provider, 'console_template')
+
+
+# When we want to setup default remote entities for provider
+# ----------------------------------------------------------
+@pytest.fixture(scope="function")
+def setup_remote_entities(provider):
+    provider.setup_test_entities_on_provider()
+
+
+@pytest.fixture(scope="function")
+def teardown_remote_entities(provider):
+    provider.teardown_test_entities_on_provider()
+# ----------------------------------------------------------
```

Comment:
**REQUIRED**
I think it's not the best place for these fixtures. Please put them into ` cfme/tests/networks/provider/test_events.py`

Reply:
Done